UI/src/html/settings.html

```diff
@@ -88,7 +88,21 @@ <h3>Name</h3>
               <div class="icon-input-container">
                 <input class="icon-input aInput" type="text" placeholder="Name" id="pointsNewName" style="background-color: #1b2e4b;color: aquamarine;border: 0px;">
             </div>
-              
+              <h2>Moderation</h2>
+              <hr style="border-color: aliceblue;">
+              <h3>Chat Filter</h3>
+              <p>Filters some bad words from the chat. No filter is perfect, this is not a substitute for a mod. Triggering the filter will cause the user to be timed out for 5 minutes.</p>
+              <div class="row">
+                <div class="col-1">
+                  <label class="switch">
+                    <input type="checkbox" id="filterEnabled">
+                    <span class="slider round"></span>
```

Comment:
Does this slider do anything? Or is it for the future where you can make the timeout dynamic?

Reply:
Just realised it's the bootstrap toggle! Not an actual "slider"